test/manifold_test.cpp

```diff
@@ -505,3 +505,39 @@ TEST(Manifold, MirrorUnion) {
   EXPECT_FLOAT_EQ(vol_a * 2.75, result.GetProperties().volume);
   EXPECT_TRUE(a.Mirror(glm::vec3(0)).IsEmpty());
 }
+
+TEST(Manifold, BooleanVolumes) {
+  glm::mat4 m = glm::translate(glm::mat4(1.0f), glm::vec3(1.0f));
+
+  // Define solids which volumes are easy to compute w/ bit arithmetics:
+  // m1, m2, m4 are unique, non intersecting "bits" (of volume 1, 2, 4)
+  // m3 = m1 + m2
+  // m7 = m1 + m2 + m3
```

Comment:
Excellent tests, thank you! Technically these should go in `boolean_test.cpp` now that I've separated them.